coc/abc.py

```diff
@@ -193,20 +193,22 @@ def __eq__(self, other):
     def __hash__(self):
         return hash((self.__name, self.__level, self.__village, self.__is_active))
 
+
     @classmethod
-    def _load_json_meta(cls, json_meta, id, name, lab_to_townhall):
+    def _load_json_meta(cls, json_meta: dict, id, name: str, lab_to_townhall):
```

Comment:
Can we maybe add here some documentation for the future outlining the structure of json_meta?